portafly/src/i18n/locales/en/shared.yml

```diff
@@ -69,25 +69,32 @@ bulk_selector:
   toggle_checkbox_aria_label: Select All
   toggle_checkbox_aria_label_description: Aria label for select all toggle button
 modals:
+  # ------------- Duplicated block: already included in Audience > Accounts > listing.yml -----·#
   send_email:
     title: Send email to admins of selected accounts
     to: To
     subject: Subject
     body: Body
     send: Send
+    cancel: Cancel  #-----duplicated string
   change_state:
     title: Change state for the selected accounts
     to: Selected accounts
     select_label: Select state
     select_helper_text: Changing the state of a developer account does not imply changing the state of its applications.
     send: Change state
+    cancel: Cancel  #-----duplicated string
+  # ------------- End of duplicated block: already included in Audience > Accounts > listing.yml -----·#
   expand_list_button: "{{count}} more"
   expand_list_button_description: Button to expand the list ot items selected for this modal
-  close_confirmation: Your content will be lost if you dismiss the dialog. Are you sure you want to dismiss the dialog?
```

Comment:
@porueesq I can't seem to find where this string ended up

Reply:
For some reason, it was removed. I'll restore it.